src/expression.d

```diff
@@ -14633,11 +14633,27 @@ extern (C++) final class PowExp : BinExp
         if (checkArithmeticBin())
             return new ErrorExp();
 
+        return rewritePowHack(this, sc);
+    }
+
+    override void accept(Visitor v)
+    {
+        v.visit(this);
+    }
+}
+
+private static Expression rewritePowHack(PowExp _pe, Scope* sc)
```

Comment:
Maybe a brief comment saying what this function does? Also, no leading _ in names, please.